nmrfx-core/src/main/java/org/nmrfx/chemistry/relax/RelaxationData.java

```diff
@@ -25,21 +25,31 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.util.*;
+import java.util.stream.Collectors;
 
 /**
  *
  * @author mbeckwith
  */
 public class RelaxationData implements RelaxationValues {
 
+    public String getId() {
+        return id;
+    }
+
```

Comment:
I think moving these methods lower down might fix a lot of the sonar lint issues, variables should be moved above methods

Reply:
I think this (and the previous, getResSource) issue is from using IntelliJ to auto add methods.  It's not very good at putting them in the right place, and I'm not very good at checking where it put them.  I'll move them.